src/platform/monitoring/DowntimeNotification/actions/index.js

```diff
@@ -54,25 +50,10 @@ export function dismissDowntimeWarning(appTitle) {
   };
 }
 
-export function getScheduledDowntime(downtimeWindow = scheduledDowntimeWindow) {
+export function getScheduledDowntime() {
   return async dispatch => {
     dispatch({ type: RETRIEVE_SCHEDULED_DOWNTIME });
 
-    // create global downtime data if feature toggle is enabled and if
-    // current time is in downtime window
-    // default to empty array
-    const { downtimeStart, downtimeEnd } = downtimeWindow;
```

Comment:
oddly removing all of this code did not break any of our unit tests.

Reply:
If I remember correctly, I think it is because you skipped some of the added tests.